uint8 role
uint8 id
LinktrackNode0[] node
